src/redist/targets/packaging/windows/clisdk/generatemanifestsmsi.ps1

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+param(
```

Comment:
I don't like how we pipe these Wix calls through PowerShell scripts.  I think it would be a lot more debuggable and easier to maintain if it was implemented in an MSBuild task or target.  However generating this MSI in this repo is only a temporary thing, so I don't care too much how it's implemented.

Reply:
Yup, wix has targets/tasks for these, if you use the .wixproj files. We're effectively reimplementing their build systems